source/learning-wazuh/vuln-detection.rst

```diff
@@ -326,19 +326,19 @@ The results should look like this:
       [root@wazuh-manager centos]# sed -i 's/config.experimental_features  = false/config.experimental_features  = true/g' /var/ossec/api/configuration/config.js
 
 
-5. Restart the Wazuh API service:
+5. Restart the Wazuh API using the Wazuh-manager service:
```

Comment:
I see a reference to config.js 3 lines above this one, please review this part deeper. Look for config.js and nodejs/npm references in the entire documentation.